api/src/main/java/org/apache/cloudstack/api/command/user/vm/UpdateVmNicIpMtuCmd.java

```diff
@@ -101,11 +105,18 @@ public Long getNicId() {
     }
 
     public String getIpaddress () {
-        if (ipAddr != null) {
-            return ipAddr;
-        } else {
+        if (Strings.isNullOrEmpty(ipAddr)) {
             return null;
         }
+        return ipAddr;
+    }
+
+    public int getMtu () {
```

Comment:
just retrun mtu will do, unless you are meaning to check for mtu <= 0 or something else.